app/database/models.py

```diff
@@ -50,6 +50,7 @@ class Event(Base):
 
     owner = relationship("User")
     owner_id = Column(Integer, ForeignKey("users.id"))
+    invitees = Column(String)
```

Comment:
Please move this line up (after line 49).
Also, what is the diff between invitees  and participants?

Reply:
It's just the email addresses of the ones invited, whether they are going to participate or not, or event if they exist as users